docs/en/v0.3/user-guide/write-data/sdk-libraries/go.md

```diff
@@ -23,31 +23,32 @@ then, you can add one `Series` into `Metric`, then create an `InsertRequest` and
 
 ```go
 func Insert() {
-    series := greptime.Series{}              // Create one row of data
-    series.AddStringTag("host", "localhost") // add index column, for query efficiency
-    series.AddFloatField("cpu", 0.90)        // add value column
-    series.AddIntField("memory", 1024)       // add value column
-    series.SetTimestamp(time.Now())          // requird
-
-    metric := greptime.Metric{} // Create a Metric and add the Series
-    metric.AddSeries(series)
-    // metric.SetTimePrecision(time.Second)  // default is Millisecond
-    // metric.SetTimestampAlias("timestamp") // default is 'ts'
-
-
-    // Create an InsertRequest using fluent style
-    // the specified table will be created automatically if it's not exist
-    insertRequest := greptime.InsertRequest{}
-    // if you want to specify another database, you can specify it via: `WithDatabase(database)`
-    insertRequest.WithTable("monitor").WithMetric(metric) // .WithDatabase(database)
-
-    // Fire the real Insert request and Get the affected number of rows
-    n, err := client.Insert(context.Background(), insertRequest)
-    if err != nil {
-        fmt.Printf("fail to insert, err: %+v\n", err)
-        return
-    }
-    fmt.Printf("AffectedRows: %d\n", n)
+	series := greptime.Series{}
+	series.AddStringTag("host", "localhost")
+	series.AddFloatField("cpu", 0.90)
+	series.AddIntField("memory", 1024)
+	series.SetTimestamp(time.Now())
+
+	metric := greptime.Metric{}
+	metric.AddSeries(series)
+
+	monitorReq := greptime.InsertRequest{}
+	monitorReq.WithTable("monitor").WithMetric(metric)
+
+	monitorCopyReq := greptime.InsertRequest{}
+	monitorCopyReq.WithTable("monitor_copy").WithMetric(metric)
```

Comment:
maybe it's better to omit the multiple table examples in greptime docs. The detail usage can be organized in greptimedb-client-go, users can find more examples there. While in docs, it's better to just tell user the whole procedure, and help user to get a correct way to interact with GreptimeDB.